core/capabilities/ccip/ccip_integration_tests/helpers.go

```diff
@@ -704,6 +723,7 @@ func wireOffRamp(t *testing.T, uni onchainUniverse, universes map[uint64]onchain
 		offrampSourceChainConfigArgs = append(offrampSourceChainConfigArgs, evm_2_evm_multi_offramp.EVM2EVMMultiOffRampSourceChainConfigArgs{
 			SourceChainSelector: getSelector(remoteChainID), // for each destination chain, add a source chain config
```

Comment:
Also misleading comment, maybe:

```suggestion
			SourceChainSelector: getSelector(remoteChainID), // for each source chain, add a source chain config
```

Though seems kinda redundant so I'd say just remove it